### ISSUE TITLE
Permitted Attributes section in Customizing Admin Guide

### ISSUE THREAD

**Issue by kennyadsl** [solidusio/solidus#3307]
At the moment we are not documenting in guides how to add extra permitted attributes to existing Solidus resources. For example, if we add a new `seo_description` column to `Spree::Taxon`, we'd also need to add the following line into an initializer to allow setting this field in the admin:

```
Spree::PermittedAttributes.taxon_attributes << :seo_description
```

**Restart the rails server after this change**

Of course, we'd also need to change the admin UI to add this field in the related form.

**Comment by octoxan** [comment]
I'll make a PR to update the docs after work if nobody else has by then. Will be my first time contributing so fingers crossed haha.

**Comment by zabrador** [comment]
Looks like this was solved by #3360 and can be closed 🎉
